cli.rkt

```diff
@@ -30,31 +32,50 @@
 (define-record-type resyntax-options (targets suite))
 
 
+;; Remove the first argument of the command line arguments
+(define-syntax-parse-rule (shift-command-line-arguments  body ...)
+  (λ args
+    (parameterize ([current-command-line-arguments (vector-copy (current-command-line-arguments) 1)])
+      body ...)))
+
+
+;; If the command line arguments are empty, re-parameterize it to
+;; default to #("--help")
+(define-syntax-parse-rule (parameterize-help-if-empty-ccla body ...)
```

Comment:
I think we ought to try to add this to `command-line` as an opt-in feature. So that we could write something like:

```scheme
(command-line
  #:program-name "foo"
  #:print-help-by-default
  flags ...)
```

Reply:
That would certainly be useful.